### PR TITLE
Renaming uap10.1 to uap10.0.15138

### PR DESCRIPTION
**Pull request by joperezr**
cc: @ericstj @weshaggard @zamont @danmosemsft 
FYI: @joshfree 

- Renaming uap10.1 to uap10.0.15138
- Big refactoring so that when we update this version again, we can just change in one place.

### CODE REVIEW

**Comment by joperezr** [comment]
FYI: @tijoytom @botaberg this will likely break the injestion of corefx into PN build, since we are changing the moniker so FYI since you will probably need to update the moniker used.

**Comment by joperezr** [comment]
FYI: @dagood @zhenlan @shmao since we will have to do similar changes in dotnet/wcf repo

File: dir.targets [hunk 1]
```diff
@@ -92,8 +92,8 @@
   <Import Project="$(MSBuildThisFileDirectory)illink.targets" />
 
   <ItemGroup Condition="'$(NuGetTargetMoniker)'=='.NETStandard,Version=v1.7'">
-    <!-- Temporarily suppress the message until we get a nuget version that knows about the mapping between netstandard1.7 and uap10.1 -->
-    <SuppressPackageTargetFrameworkCompatibility Include="uap10.1" />
+    <!-- Temporarily suppress the message until we get a nuget version that knows about the mapping between netstandard1.7 and uapvNext -->
```
Comment (danmosemsft) [inline dir.targets]: Do we have something in nuget repo tracking this?

Comment (joperezr) [inline dir.targets]: We do, @ericstj logged an issue some time ago, but we don't have the final moniker yet, so we don't want them to fix it just yet. This moniker we are going to use now will unblock @zamont 's testing, but it's not the final TFM to be used yet, we will still have to change it before shipping.

**Pull request merged**